bookkeeper-server/src/test/java/org/apache/bookkeeper/client/BookieDecommissionTest.java

```diff
@@ -88,7 +98,7 @@ public void testDecommissionBookie() throws Exception {
          */
         bkAdmin.decommissionBookie(BookieImpl.getBookieId(killedBookieConf));
         bkAdmin.triggerAudit();
-        Thread.sleep(500);
+        Thread.sleep(5000);
```

Comment:
out of the scope of this PR, but in the future it would be better to wait for a specific condition, in order to reduce flakyness